bundler/lib/bundler/man/gemfile.5.ronn

```diff
@@ -580,6 +580,7 @@ When determining path to the lockfile or whether to create a lockfile, the
 following precedence is used:
 
 1. The `bundle install` `--no-lock` option (which disables lockfile creation).
+1. The `bundle install` `--lockfile` option.
 2. The `lockfile` method in the Gemfile.
 3. The `BUNDLE_LOCKFILE` environment variable.
 4. The default behavior of adding `.lock` to the end of the Gemfile name.
```

Comment:
Should we use `1.` for all items? Or should we renumber them?

Reply:
Good catch. Probably use `1.` for everything to avoid future issues like this. I'll push an updated commit.